source/regression2.md

```diff
@@ -274,7 +274,7 @@ for i in range(len(slope_l)):
             )
         )
         .mark_line(color=line_color_l[i])
-        .encode(x=alt.X("x"), y=alt.Y("y"))
+        .encode(x="x", y="y")
```

Comment:
We should be as consistent as possible everywhere -- do we use `(x="x", y="y")` everywhere, or `alt.X("x")` etc everywhere? should pick one or the other and stick with it, ideally, assuming that's possible